api/src/Doctrine/Filter/ExpressionDateFilter.php

```diff
@@ -45,14 +45,14 @@ public function getDescription(string $resourceClass): array {
      */
     protected function filterProperty(
         string $property,
-        $values,
+        $value,
```

Comment:
Warum wollen sie hier nicht mehr values?

Reply:
AbstractFilter definiert in der Signatur für filterProperty `$value` und Phpstan (oder Psalm?) möchte, dass man beim überschreiben von Methoden die Signatur gleich lässt (Anzahl, Type, Name).